index.html

```diff
@@ -0,0 +1,189 @@
+                                <li class="bootstrap-tag"><a href="#">bootstrap</a></li>
+                                <li class="ruby-tag"><a href="#">ruby</a></li>
+                            </ul>
+                        </nav>
+                    </div>
+                    <button class="see-project-btn1">See Project</button>
+                </div>
+                <div class="card-works">
+                    <div class="frame43-right-block">
+                        <h1 class="title-post1">
+                            Profesional Art Printing Data
+                        </h1>
+                        <p class="supporting-text1">
+                            A daily selection of privately personalized reads; no accounts or sign-ups required. has
+                            been the industry's standard
+                        </p>
+                        <nav>
+                            <ul class="tags1">
+                                <li class="html-tag"><a href="#">html</a></li>
+                                <li class="bootstrap-tag"><a href="#">bootstrap</a></li>
+                                <li class="ruby-tag"><a href="#">ruby</a></li>
+                            </ul>
+                        </nav>
+                    </div>
+                    <button class="see-project-btn1">See Project</button>
+                </div>
+                <div class="card-works" id="last">
+                    <div class="frame43-right-block">
+                        <h1 class="title-post1">
+                            Profesional Art Printing Data
+                        </h1>
+                        <p class="supporting-text1">
+                            A daily selection of privately personalized reads; no accounts or sign-ups required. has
+                            been the industry's standard
+                        </p>
+                        <nav>
+                            <ul class="tags1">
+                                <li class="html-tag"><a href="#">html</a></li>
+                                <li class="bootstrap-tag"><a href="#">bootstrap</a></li>
+                                <li class="ruby-tag"><a href="#">ruby</a></li>
+                            </ul>
+                        </nav>
+                    </div>
+                    <button class="see-project-btn1">See Project</button>
+                </div>
+            </div>
+        </section>
+        <section class="about-me">
+            <div class="about-me-div">
+                <h1>About me</h1>
+                <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <button class="get-my-resume">Get My Resume</button>
+            </div>
+            <div class="divider"></div>
+            <nav class="skill">
```

Comment:
- [ ] I think you might remove this `nav` tag; the `nav` element represents a section of a page whose purpose is to provide navigation links, either within the current document or to other documents. Common examples of navigation sections are menus, tables of contents, and indexes. ℹ️